widget/embedded/src/hooks/useConfirmSwap/useConfirmSwap.helpers.ts

```diff
@@ -26,49 +32,58 @@ import {
 /**
  * A request can be successful but in body of the response, it can be some case which is considered as failed.
  */
-export function throwErrorIfResponseIsNotValid(response: BestRouteResponse) {
-  if (!response.result) {
+export function throwErrorIfResponseIsNotValid(response: QuoteResponse) {
+  if (!response.swaps) {
     throw new Error(errorMessages().noResultError.title, {
       cause: {
         type: QuoteErrorType.NO_RESULT,
         diagnosisMessage: response.diagnosisMessages?.[0],
       },
     });
   }
-  const limitError = hasLimitError(response);
+  const quoteError = getQuoteError(response.swaps);
+  if (quoteError) {
+    throw new Error(quoteError.message, {
+      cause: quoteError.options,
+    });
+  }
+}
+
+export function getQuoteError(swaps: SwapResult[]): QuoteErrorResponse | null {
+  const limitError = hasLimitError(swaps);
   if (limitError) {
     const { swap, recommendation, fromAmountRangeError } =
-      getLimitErrorMessage(response);
-
-    throw new Error('bridge limit error', {
-      cause: {
+      getLimitErrorMessage(swaps);
+    return {
+      message: 'bridge limit error',
```

Comment:
It seems we are not showing route in bridge limit error anymore.
<img width="486" alt="Screenshot 2024-02-17 at 15 32 04" src="https://github.com/rango-exchange/rango-client/assets/99180934/54a15860-c366-45fa-bbba-42fa6cabad8f">